module-2/backend/week_three.md

```diff
@@ -10,14 +10,42 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+ - `rails new project_name --optional --flags`  
```

Comment:
👍 Like that you specify those flags as optional. Shows that you know they're available to you *and* that you don't need them.